.changesets/exp_geal_entity_cache_documentation.md

```diff
@@ -0,0 +1,21 @@
+### Entity cache preview ([PR #5574](https://github.com/apollographql/router/pull/5574))
+
+#### Private information caching
+
+When a subgraph returns a `Cache-Control: private` header, the response data should not be cached and shared among users. But since the router supports request authentication, it can use it to allocate separate cache entries per users. This is done via the `private_id` option, which configures the name of a key in the request context, that contains the data used to differentiate users. This must be paired with a coprocessor or rhai script to set the value in context.
+
+Example configuration:
```

Comment:
```suggestion
### Entity cache preview ([PR #5574](https://github.com/apollographql/router/pull/5574))

#### Support private information caching

The router supports a new `private_id` option that enables separate, private cache entries to be allocated per user for authenticated requests.

When a subgraph returns a `Cache-Control: private` header, the response data shouldn't be cached and shared among users. However, since the router supports request authentication, it can use it to allocate separate cache entries per users. 

To enable this, configure the `private_id` to be the name of a key in the request context that contains the data that's used to differentiate users. This option must be paired with a coprocessor or Rhai script to set the value in context.

Example configuration:
```